draft-ietf-tls-external-psk-importer.md

```diff
@@ -87,6 +87,47 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 - External Identity: The identity of an EPSK.
 - Imported Identity: The identity of a PSK as sent on the wire.
 
+#Binder Key
+To prevent PSK Importers from being confused with standard OOB PSKs we change the label used in the computation of the PSK binder key.
+In TLS 1.3 the  PSK binder key computation is defined as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" | "res binder", "")
+                 |                     = binder_key
+
+We replace the string "ext binder" with "imp binder".
+This means the binder key is now computed as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" 
+                                        | "res binder" 
+                                        | "imp binder", "")
+                 |                     = binder_key
+
+This change should not affect any of the securtiy properties or security proofs previously established.
+The only change that is not a previously valid TLS 1.3 trace we mostly focus our attention on the change to the `binder_key` computation.
+The new label prevents the key being used in other contexts, including for encrypting or decrypting messages produced elsewhere.
```

Comment:
Hmm... I'm not sure it does. There's nothing stopping someone from deriving a binder key in the same way and using it to encrypt or MAC something. Perhaps simply:

"Label differentiation prevents collisions with non-imported external keys."

Reply:
I guess that's true, I was thinking in terms of types, so yes you could use it to encrypt but you wouldn't get a valid result. I'll rephrase to something correct in real life :P.